.github/workflows/close-stale-feature-requests.yml

```
@@ -0,0 +1,43 @@
+name: Close stale feature requests
+on:
+  schedule:
+    - cron: 0 1 * * *
+
+env:
+  CLOSE_MESSAGE: >
+    There has been no activity on this feature request
+    and it is being closed. We value your input
+    and if you feel closing this issue was not the
+    right thing to do, please re-open it.
+
+    For more information on how the project manages
+    feature requests please consult the
+    [feature request management document](https://github.com/nodejs/node/blob/HEAD/doc/guides/feature-request-management.md).
+
+  WARN_MESSAGE: >
+    There has been no activity on this feature request for
+    5 months and it is unlikely to be implemented.
+    It will be closed after 6 months after the last non-automated comment
+    unless there is renewed discussion or collaborators advocating that
+    it be kept open. For more information on how the project manages
```

Remove redundant material:

```suggestion
    For more information on how the project manages
```